Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -5702,18 +5702,32 @@ void HTMLMediaElement::mediaPlayerSizeChanged()
     endProcessingMediaPlayerCallback();
 }
 
-bool HTMLMediaElement::mediaPlayerRenderingCanBeAccelerated()
+void HTMLMediaElement::invalidateStyleAndLayerComposition()
 {
+    HTMLElement::invalidateStyleAndLayerComposition();
+    updateAcceleratedRenderingState();
+}
+
+void HTMLMediaElement::updateAcceleratedRenderingState()
```

Comment:
Should/Can this be moved to HTMLVideoElement?

Reply:
It probably can

Reply:
moved stuff to HTMLVideoElement, updated commit log